dbt_project.yml

```diff
@@ -51,7 +55,8 @@ vars:
     snowplow__max_session_days: 3
     snowplow__upsert_lookback_days: 30
     snowplow__query_tag: "snowplow_dbt"
-    snowplow__incremental_materialization: "snowplow_incremental"
+    snowplow__incremental_materialization: "incremental"
```

Comment:
I assume we will eventually deprecate this variable completely?

Reply:
See discussion for this, undecided.